code/script.js

```diff
@@ -0,0 +1,114 @@
+      <p>Workplace: ${json.company}</p>
+      <p>Current location: ${json.location}
+      <p>Portfolio: <a href="${json.blog}">${json.blog}</a></p>
+      <p>This account has a total of ${json.public_repos} repos</p>
+      <p>Bio: ${json.bio}</p>
+      </div>
+      `;
+    });
+};
+
+// This function is used to present the repo data for my projects
+const presentRepoData = () => {
+  // fetching my data using the global variable
+  fetch(REPOS_URL) 
+    .then((response) => {
+      // this handles the json data that is fetched from the api
+      return response.json(); 
+    })
+    .then((json) => {
+      // creating a filter that filters out every repo that has been forked and starts with "project-"
+      const technigoRepos = json.filter(
+        (project) => project.fork && project.name.startsWith("project-")
+      );
+      
+      // The data that is fetched from the API is put in an array and I use my forEach loop to present every project
+      technigoRepos.forEach((project, position) => {
+        // creates a variable to use for the current project
+        let currentProject = project.name;
+        // creates a new url to fetch from where i use the current project and my user name
+        const COMMIT_COUNT_URL = `https://api.github.com/repos/${USER}/${currentProject}/commits`;
+        // fetches the data from the API with the variable above
+        // I do a fetch within the fetch because I want to use the data from the first fetch to create the url for the second fetch
+        fetch(COMMIT_COUNT_URL) 
+          .then((response) => {
+            // this return handles the json data
+            return response.json(); 
+          })
+          .then((json) => {
+            // This filter outs the commits to only show the ones that has been made by me
+            const filteredCommits = json.filter(
+              (project) => project.commit.committer.name === "Linnea Isebrink"
+            );
+            // This is the date and it gets the latest post
+            let commitDate = filteredCommits[0].commit.author.date;
+            // This is the commit message and it gets the latest
+            let commitMessage = filteredCommits[0].commit.message;
+            // I present the data with innerHTML
+            projectsContainer.innerHTML += `
+            <div class="project" id="${position}">
+            <div class="project-header"><h3>${
+              project.name
+            }</h3><span id="deleteRepo">X</span></div>
+            <div class="project-text">
+                <p>Main branch for this project is: ${
+                  project.default_branch
+                }</p>
+                <a href="${project.html_url}">${project.html_url}</a>
+                <p>There has been ${
+                  filteredCommits.length - 1
+                } commits</p>
+                <p>Commit date: ${commitDate.slice( // I use slice() to make the date look prettier
```

Comment:
Didn´t know that way to make the date easier to read, but will definitely remember it!